### PR TITLE
mainly changes to sql.c for fun

### PR DESCRIPTION
**Pull request by SCostello84**
sql.c now prints line by line to screen then prompts for user input...infinitely loops

### CODE REVIEW

**Comment by nickJones67** [comment]
I'm digging the slow scroll thing. Abbassi will love it

File: cop4710-sequel.sln [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿
```
Comment (JonathanFrias) [inline cop4710-sequel.sln]: can you list this file in `.gitignore`?

File: src/common.h [hunk 2]
```diff
@@ -84,7 +84,13 @@ struct ParseTree* parseGrammer(char* sql);
 bool storeTuple(struct Tuple* tuples, int count);
 bool storeTable(struct Table* table);
 
+<<<<<<< HEAD
+struct Table* retrieve(struct ParseTree* tree);
+struct Table* createTable(int count);
+void SqlRun();
+=======
```
Comment (JonathanFrias) [inline src/common.h]: merge conflict! Get this to compile and commit the result!

File: src/test.c [hunk 3]
```diff
@@ -120,16 +120,35 @@ int main(void) {
...
+  //printTable(table);
+  //destroyExampleTable(table);
 
+  getchar();
   printf("\n===============testStore\n");
   testStore();
+<<<<<<< HEAD
+
+  getchar();
+=======
```
Comment (JonathanFrias) [inline src/test.c]: merge conflict! Get this to compile and commit the result!

**Comment by JonathanFrias** [comment]
Try: 

``` diff
index acdd969..eb50ce1 100644
--- a/src/sql.c
+++ b/src/sql.c
@@ -24,20 +24,19 @@ int main(void)
        for (i = 0; i < strlen(sqlLine); i++)
        {
                printf("%c", sqlLine[i]);
-               usleep(10000);
+               // usleep(10000);
        }

        printf("%45s\n", "Welcome to wSQLx!");


-       char * sqlString = "wSQLx: ";
-       char * input = {0};
+       char* sqlString = "wSQLx: ";
+       char input[1000];
        do
        {
                printf("%sPlese input command.", sqlString);
                scanf("%s", input);

-
-       }while (input != "exit");    //this infinitely loops
+       }while (strcmp(input, "exit") != 0);
 }

```

notice how `input` is a pointer to **valid storage space**, which is required for scanf to put stuff into! 
`scanf()` was was following the pointer, and writing bytes but it didn't actually reference any valid storage! Hence seg fault.

**Comment by SCostello84** [comment]
That wasn't the problem. I did a bit more on sql.c since that push. The one
I pushed was just to get the wsqlx banner passed on

**Comment by JonathanFrias** [comment]
Okay that's fine. I thought that initially, but the merge conflicts + extra code cause a little bit of confusion!

**Pull request merged**